src/database/query-table.c

```diff
@@ -662,7 +706,7 @@ void DB_read_queries(void)
 	// Get time stamp 24 hours in the past
 	const time_t now = time(NULL);
 	const time_t mintime = now - config.maxlogage;
-	const char *querystr = "SELECT * FROM queries WHERE timestamp >= ?";
+	const char *querystr = "SELECT id,timestamp,type,status,domain,client,forward,additional_info,reply_type,reply_time,dnssec FROM queries WHERE timestamp >= ?";
```

Comment:
In case the database is further extended in the future, this ensures we only source those fields we know. Also, this ensures we get the on the order we expect them. Until now, we just relied on the database reporting them in the order of their definition (which seems logic, too). It is no extra work for SQLite3 only the initial byte compilation process is a bit longer but this is done only once over the entire lifetime of FTL.